inc/saitypes.h

```diff
@@ -555,6 +555,7 @@ typedef union {
     sai_ip4_t ip4;
     sai_ip6_t ip6;
     sai_ip_address_t ipaddr;
+    sai_ip_prefix_t ipprefix;
```

Comment:
the concern is that this doubles the sai_attribute_value size. for ipv6, the attribute value size will be 16*4 = 64 bytes.